Client/Extensions/UIImageViewExtensions.swift

```diff
@@ -85,45 +85,3 @@ open class ImageOperation: NSObject, SDWebImageOperation {
     }
 }
 
-// This is an extension to SDWebImage's api to allow passing in a cache to be used for lookup.
-public typealias CompletionBlock = (_ img: UIImage?, _ err: NSError, _ type: SDImageCacheType, _ key: String) -> Void
-extension UIImageView {
-    // This is a helper function for custom async loaders. It starts an operation that will check for the image in
-    // a cache (either one passed in or the default if none is specified). If its found in the cache its returned,
-    // otherwise, block is run and should return an image to show.
-    fileprivate func runBlockIfNotInCache(_ key: String, cache: SDImageCache, completed: @escaping CompletionBlock, block: @escaping () -> UIImage?) {
-        self.sd_cancelCurrentImageLoad()
-
-        let operation = ImageOperation()
-
-        operation.cacheOperation = cache.queryDiskCache(forKey: key, done: { (image, cacheType) -> Void in
-            let err = NSError(domain: "UIImage+Extensions.runBlockIfNotInCache", code: 0, userInfo: nil)
-            // If this was cancelled, don't bother notifying the caller
-            if operation.cancelled {
-                return
-            }
-
-            // If it was found in the cache, we can just use it
-            if let image = image {
-                self.image = image
-                self.setNeedsLayout()
-            } else {
-                // Otherwise, the block has a chance to load it
-                let image = block()
-                if image != nil {
-                    self.image = image
-                    cache.store(image, forKey: key)
-                }
-            }
-
-            completed(image, err, cacheType, key)
-        })
-
-        self.sd_setImageLoadOperation(operation, forKey: "UIImageViewImageLoad")
-    }
-
-    public func moz_getImageFromCache(_ key: String, cache: SDImageCache, completed: @escaping CompletionBlock) {
-        // This cache is filled outside of here. If we don't find the key in it, nothing to do here.
-        runBlockIfNotInCache(key, cache: cache, completed: completed) { _ in return nil}
-    }
-}
```

Comment:
Was this all dead code?

Reply:
Either dead or deeply resting. No one called it that I could find.

Reply:
👍 💯